Week2/Section2_w2/MySpotify/Song.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Song.h"
+#include <string>
+
+Song::Song()
+{
+	SongTitle = "";
+	ArtistName = "";
+	Duration = 0;
+}
+
+Song::Song(const std::string NewSongTitle, const std::string NewArtistName, const int NewDuration)
+{
+	SongTitle = NewSongTitle;
+	ArtistName = NewArtistName;
+	Duration = NewDuration;
+}
+
+std::string Song::GetFormatedDuration()
```

Comment:
This method could be const. It is not modifying the object in any way, it's just reading its values.